plugins/modules/aci_rest.py

```diff
@@ -413,12 +414,17 @@ def main():
     aci.method = aci.params.get('method').upper()
 
     # Perform request
-    resp, info = fetch_url(module, aci.url,
-                           data=payload,
-                           headers=aci.headers,
-                           method=aci.method,
-                           timeout=aci.params.get('timeout'),
-                           use_proxy=aci.params.get('use_proxy'))
+    resp = None
+    if module._socket_path:
+        conn = Connection(aci.module._socket_path)
+        info = conn.send_request(aci.method, '/' + path, payload)
```

Comment:
let's use format as we generally use so something like : '/{0}'.format(call_path)